Tests/AssetTest.swift

```diff
@@ -15,13 +15,13 @@ class AssetTest: XCTestCase {
     }
     #if !NO_FATAL_TEST
 //    The test runner exited with code 6 before finishing running tests. This may be due to your code calling 'exit', consider adding a symbolic breakpoint on 'exit' to debug.
-    func testFetch_withoutUID() async {
-        expectFatalError(expectedMessage: "Please provide Asset uid") {
-            makeAssetSut().fetch { (result: Result<AssetModel, Error>, response) in
-                        
-            }
-        }
-    }
+//    func testFetch_withoutUID() async {
```

Comment:
@vkalta Could you please clarify the reason for commenting out the test cases block?

Reply:
This is outdated code @sunil-lakshman . This needs to run in the main thread else an error is thrown. So will revisit it later.